### PR TITLE
Cause of Injury

### PR DESCRIPTION
**Pull request by JimmySunDance**

### CODE REVIEW

File: schemas/reference/bulkload/injurycause.csv [hunk 1]
```diff
@@ -0,0 +1,22 @@
+id,injurycause,description,validfrom,validto
+1,Harmful chemicals / substances,"Asbestos, irritants, corrosive liquids, etc",2020-04-17 00:01:00+00,
+2,Contact / collision: Fixed object,Collision with a static object.,2020-04-17 00:01:00+00,
+3,Contact / collision: Person,Collision with another person.,2020-04-17 00:01:00+00,
+4,Contact / collision: Moving object,"Collision with a moving object; vehicle, transport etc.",2020-04-17 00:01:00+00,
+5,Electricution: faulty equiptment / machinery,Electricution caused by broken / damaged equiptment.,2020-04-17 00:01:00+00,
```
Comment (belindac-vd) [inline schemas/reference/bulkload/injurycause.csv]: faulty equipment, not equiptment
damaged equipment, equiptment

Comment (belindac-vd) [inline schemas/reference/bulkload/injurycause.csv]: All electrocution is spelt incorrectly, not electricution

File: schemas/reference/bulkload/injurycause.csv [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+6,Electricution: non-faulty equiptment,Electricution through miss hangling equiptment.,2020-04-17 00:01:00+00,
```
Comment (belindac-vd) [inline schemas/reference/bulkload/injurycause.csv]: All equipment seems to be spelt incorrectly equiptment

File: schemas/reference/bulkload/injurycause.csv [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+7,Equipment / tools: Not fit for task,The tool used was not fit for the task being performed.,2020-04-17 00:01:00+00,
+8,Equipment / tools: Operator error,Error in the operation of the tool in use.,2020-04-17 00:01:00+00,
+9,Equipment / tools: Faulty,Operation of faulty equiptment.,2020-04-17 00:01:00+00,
+10,Slip / trip / fall,,2020-04-17 00:01:00+00,
+11,Asphixiation: Chocking / strangulation,Asphixiation through obstruction of airway.,2020-04-17 00:01:00+00,
+12,Asphixiation: Fume inhalation / drowning,Asphixiation through lack of availible oxygen.,2020-04-17 00:01:00+00,
```
Comment (belindac-vd) [inline schemas/reference/bulkload/injurycause.csv]: available, not availible

File: schemas/reference/bulkload/injurycause.csv [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+13,Falling object / projectile,,2020-04-17 00:01:00+00,
+14,Hot / cold: Surface / liquid / gas,,2020-04-17 00:01:00+00,
+15,Moving / handling: manual lifting / pushing / pulling,,2020-04-17 00:01:00+00,
+16,Moving / handling: Use of lifting aid i.e. trolley,,2020-04-17 00:01:00+00,
+17,Personal safety training / techniques (PST),,2020-04-17 00:01:00+00,
+18,Posture / bending / stretching,,2020-04-17 00:01:00+00,
+19,Sharp object: Needles / knives etc,,2020-04-17 00:01:00+00,
+20,Falling from height,Where the subject has been working at hight and fallen,2020-04-17 00:01:00+00,
```
Comment (belindac-vd) [inline schemas/reference/bulkload/injurycause.csv]: height, not hight

File: schemas/reference/bulkload/injurycause.csv (hunk 2 above)
Comment (Ben-UK) [inline schemas/reference/bulkload/injurycause.csv:7]: spelling and spacing errors: faulty equipment / machinery, Electrocution caused by broken / damaged equipment.,

File: schemas/reference/bulkload/injurycause.csv (hunk 1 above)
Comment (Ben-UK) [inline schemas/reference/bulkload/injurycause.csv:6]: Spelling and spacing errors: "Electrocution: faulty equipment / machinery, Electrocution caused by broken / damaged equipment"

File: schemas/reference/bulkload/injurycause.csv [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+2,Contact / collision: Fixed object,Collision with a static object.,2020-04-17 00:01:00+00,
+3,Contact / collision: Person,Collision with another person.,2020-04-17 00:01:00+00,
+4,Contact / collision: Moving object,"Collision with a moving object; vehicle, transport etc.",2020-04-17 00:01:00+00,
+5,Electricution: faulty equiptment / machinery,Electricution caused by broken / damaged equiptment.,2020-04-17 00:01:00+00,
+6,Electricution: non-faulty equiptment,Electricution through miss hangling equiptment.,2020-04-17 00:01:00+00,
+7,Equipment / tools: Not fit for task,The tool used was not fit for the task being performed.,2020-04-17 00:01:00+00,
+8,Equipment / tools: Operator error,Error in the operation of the tool in use.,2020-04-17 00:01:00+00,
+9,Equipment / tools: Faulty,Operation of faulty equiptment.,2020-04-17 00:01:00+00,
+10,Slip / trip / fall,,2020-04-17 00:01:00+00,
+11,Asphixiation: Chocking / strangulation,Asphixiation through obstruction of airway.,2020-04-17 00:01:00+00,
```
Comment (Ben-UK) [inline schemas/reference/bulkload/injurycause.csv]: Asphyxiation

File: schemas/reference/bulkload/injurycause.csv (hunk 3 above)
Comment (Ben-UK) [inline schemas/reference/bulkload/injurycause.csv]: "Asphyxiation through lack of available oxygen"

File: schemas/reference/bulkload/injurycause.csv (hunk 4 above)
Comment (Ben-UK) [inline schemas/reference/bulkload/injurycause.csv]: height

**Pull request merged**